packages/mcl/src/src/mcl/utils/coda.d

```diff
@@ -527,6 +527,16 @@ struct CodaApiClient
         coda.deleteRow("dEJJPwdxcw", tables[0].id, resp[0]);
     }
 
+    void updateOrInsertRow(string docId, string tableId, RowValues values) {
```

Comment:
According to the Coda Api docs, we should be able to directly create an "Upsert" request, without first fetching the data:

https://coda.io/developers/apis/v1#tag/Rows/operation/upsertRows

Can you look into directly doing an upsert?